make_html.py

```diff
@@ -22,12 +22,25 @@
 github_token = os.environ.get("GITHUB_TOKEN")
 mathlib4repo = github.Github(github_token).get_repo("leanprover-community/mathlib4")
 
+
 @functools.cache
 def github_labels(pr):
     pull_request = mathlib4repo.get_pull(pr)
-    labels = [{'name': label.name, 'color': label.color} for label in pull_request.get_labels()]
+
+    def text_color_of_color(color):
+        r, g, b = map(lambda i: int(color[i:i + 2], 16), (0, 2, 4))
+        perceived_lightness = (
+            (r * 0.2126) + (g * 0.7152) + (b * 0.0722)) / 255
+        lightness_threshold = 0.453
+        return 'black' if perceived_lightness > lightness_threshold else 'white'
```

Comment:
Note that while this presumably matches the github light mode behavior, it does not match the dark mode behavior

Reply:
Probably. I only looked superficially at what GitHub is doing exactly. However I think it is an improvement over what we currently have even in the dark mode